### DISCUSSION TITLE
I need a tip/help to solve a work issue using QField

### DISCUSSION THREAD

**Discussion by Lighffer** [opengisch/QField#5790] [Q&A]
First of all, sorry if this is not the right place to ask this kind of question.

I am from a government agency that works with rural property registration, and I use QGIS for the georeferencing of properties in each city of my state.

However, I need a way to use QField to migrate a single overall project to the tablet that will be used in the field services, where I can update it regularly and the updates will reach the other colleagues in the field.

Since there are many people involved, I believe it would be unfeasible for the agency to pay for QField PRO for each unit.

If there are other ways to solve this problem more easily and you can assist me with that, I would appreciate it.

**Comment by ZsanettMed** [comment]
Hi @Lighffer 

Thank you for reaching out. We would like to hear more details about your project. Please contact us at [sales@qfield.cloud](mailto:sales@qfield.cloud). 

Awaiting your email.
Best,
Zsanett
